packages/services/src/utils/helpers.ts

```diff
@@ -257,3 +257,8 @@ export const constructObject = (
 
   return data;
 };
+
+export const extractDomain = (url: string): string => {
```

Comment:
From my understanding, if we do not have try/catch when calling `new URL`, the default error is `Throws TypeError: Invalid URL`. Should we custom this error?

Reply:
No, we shouldn't